### ISSUE TITLE
Linux No audio passtrough with internal keying on DeckLink 4K SDI

### ISSUE THREAD

**Issue by walterav1984** [CasparCG/Server#602]
Although #370 came with a workaround and is fixed for both 2.07 and 2.1beta2 CasparCG Server, just tested it in Windows 10 15063 (BMD driver 10.8.6), this problem (still) exists for the Linux CasparCG 2.1beta2 build but might be a Blackmagic Desktop video Driver issue?

Can anyone confirm when using internal keying on a Decklink the audio from the SDI input on which CasperCG is overlaying its key+fill is absent on the SDI out, however CasparCG its own audio within the key feed is playing fine on the SDI out. 

```
        <consumers>
            <decklink>
                <device>1</device>
                <embedded-audio>true</embedded-audio>
                <channel-layout>stereo</channel-layout>
                <latency>normal</latency>
                <keyer>internal</keyer>
                <key-only>false</key-only>
                <buffer-depth>3</buffer-depth>
            </decklink>
        </consumers>
```

Its workaround (opening BMD Media Express) during or before CasparCG server is initialized is not valid for bringing back the sound.

Hardware: Decklink 4K SDI driver 10.8.6
Software: CasparCG 2.1b2 ubuntu 16.04.2 kernel 4.4.x72

**Comment by jesperstarkar** [comment]
If so, I'm certain this is a BMD driver bug - as you suggest.

**Comment by walterav1984** [comment]
BMD support has been mailed...

**Comment by walterav1984** [comment]
Seems fixed in Desktop Video 10.9, I'll close for now.
